src/components/Order.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import Select from 'react-select';
+
+import { useDispatch } from 'react-redux';
+import { displayTransaction } from '../redux/transaction/TransactionReducer';
+
+const Order = () => {
+  const [order, setOrder] = useState(' asc ');
+  const dispatch = useDispatch();
+  const options = [
+    {
+      id: 1,
+      value: 'asc',
+      label: 'Ascending',
+    },
+    {
+      id: 2,
+      value: 'desc',
+      label: 'Descending',
+    },
+  ];
+
+  const handleChange = (e) => {
+    console.log(e, e.value);
+    setOrder(e.value);
+    if (e.value === 'asc') {
+      dispatch(displayTransaction({ order: 'asc' }));
+    }
+    dispatch(displayTransaction({ order: 'desc' }));
+  };
```

Comment:
![Screen Shot 2022-09-02 at 12 14 35 PM](https://user-images.githubusercontent.com/83717285/188204961-09633127-e7f0-4f9a-839c-3bafe3c6af06.png)


- [ ] Your app is looking great, excellent job! However, the 'order by:' feature is a sorting method but not a filter as the activity requires:

- You should filter the data that you retrieve from the API using a Filter stateless component.

It would be great to add a filtering method, e.g: Filter by Value. Or any other filtering method that meets the activity requirement.

Reply:
Hi @smunozmo , thank you for pointing this out, I have fixed this issue. I used the if the approach to filter the ascending and descending order. These other comes directly from the API, so I made the API parameter to sort them, as a variable and passed either ascending or descending order to the API, to filter out the data. I have also updated the demo video as instructed.

Thanks.